### ISSUE TITLE
No Canada Flag?

### ISSUE THREAD

**Issue by iamliammckimm** [dhanishgajjar/css-flags#1]
Canada is very big important country, please add

**Comment by dhanishgajjar** [comment]
Hey @iamliammckimm Thanks, not just Canada, many countries including United States, India, New Zealand and Australia were left out, because at this moment I haven't figured out a way to make them using just one single div. But hopefully will be adding soon, can't promise how soon :)

**Comment by gusbemacbe** [comment]
@dhanishgajjar, check: https://codepen.io/JDHorn/pen/oIvcu . ;-)

**Comment by dhanishgajjar** [comment]
@gusbemacbe also this one, not a single div flag

**Comment by gusbemacbe** [comment]
If you want to figure how to draw a complicated shape in the single div, you need to use background hacks, check http://lea.verou.me/css3patterns/

**Comment by gusbemacbe** [comment]
Maybe @lea_vrou knows.

**Comment by gusbemacbe** [comment]
@LeaVrou

**Comment by dhanishgajjar** [comment]
@gusbemacbe Yes I know :) that link .. still not all shapes are possible ... give it a try may be?

**Comment by gusbemacbe** [comment]
I'll test afternoon. But @LeaVrou knows better than me.

**Comment by gusbemacbe** [comment]
Have you  already seen https://hacks.mozilla.org/2014/09/single-div-drawings-with-css/ before?

**Comment by gusbemacbe** [comment]
Here is the https://stackoverflow.com/questions/48019931/css-single-div-drawing-shapes

This guy asked how to draw a complicated eagle in the single div.

**Comment by dhanishgajjar** [comment]
@gusbemacbe Thanks, but I am done with this project. If you look at the flags css files, you'll see I use the same background property for these flags .. I know all what you are sending me :) I appreciate it. but I am done with this project long time ago.
